java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2791,6 +2791,38 @@ public void testToString() {
     }
   }
 
+  @Test
+  public void testUInt1VectorToString() {
+    try (final UInt1Vector uInt1Vector = new UInt1Vector("uInt1Vector", allocator)) {
+      setVector(uInt1Vector, (byte) 0xff);
+      assertEquals("[255]", uInt1Vector.toString());
+    }
+  }
+
+  @Test
+  public void testUInt2VectorToString() {
+    try (final UInt2Vector uInt2Vector = new UInt2Vector("uInt2Vector", allocator)) {
+      setVector(uInt2Vector, (char) 0xffff);
+      assertEquals("[\uffff]", uInt2Vector.toString());
```

Comment:
actually, this doesn't look write. weshould convert this to the integer representation I think.

Reply:
I see your point. Please check if it looks right now?